lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -50,6 +50,7 @@ def check_config(self, client, server_config, host, my_headers={}):
                 "results": f"{host}/results",
                 "elasticsearch": f"{uri}/elasticsearch",
                 "endpoints": f"{uri}/endpoints",
+                "index_search": f"{uri}/index/search",
```

Comment:
I'd prefer that you put this with `"index_mappings"` to keep related entry points together. Functionally it doesn't really matter, but there's no reason to make this list random. (Or maybe we should just alphabetize it, which is what you'd see if you looked at `window.endpoints` in the UI.)